misc/config_tools/schema/checks/vm_types.xsd

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<xs:schema xml:id="root"
+	   xmlns:xs="http://www.w3.org/2001/XMLSchema"
+	   xmlns:acrn="https://projectacrn.org">
+
+  <xs:assert test="count(/acrn-config/vm[vm_type = 'SERVICE_VM']) &lt; 2">
+    <xs:annotation acrn:severity="error">
+      <xs:documentation>There can be at most one service VM, but more than one was configured. Verify there is only one VM with  vm_type set to SERVICE_VM. </xs:documentation>
```

Comment:
How does one get more than one Service VM Can someone change the vm_type to SERVICE_VM for any VM in the configurator, and this is the assert that will catch that error (eventually)?

Reply:
Users cannot hit this when using the configurator.

Meanwhile, these checks are also applied at an early stage of building, so that errors in manually-edited XMLs can be caught early. VM type errors could exist in this case.

Reply:
Good you're thinking of errors that could be introduced by editing the xml file. Have you considered the configurator adding a test to detect manual edits (hash or checksum kind of test)?